frontend/src/component/common/Markdown/Markdown.tsx

```diff
@@ -22,3 +22,7 @@ const LinkRenderer = ({
 export const Markdown = (props: ComponentProps<typeof ReactMarkdown>) => (
     <ReactMarkdown components={{ a: LinkRenderer }} {...props} />
 );
+
+export const SimpleMarkdown = (props: ComponentProps<typeof ReactMarkdown>) => (
+    <ReactMarkdown components={{ a: LinkRenderer }} skipHtml allowedElements={['a', 'p', 'strong', 'em']} {...props} />
+);
```

Comment:
Not sure if it's better but I wonder if you could skip the new dependency by creating a PlainTextMarkdown component that allows/renders no elements, just the plain text.

Reply:
Nvm, I see how the plain text is useful for the highlighting, so I prefer your approach.